packages/components/src/index.ts

```diff
@@ -8,10 +8,6 @@ export * from "./core/Alert";
 export { default as Alert } from "./core/Alert";
 export * from "./core/Autocomplete";
 export { default as Autocomplete } from "./core/Autocomplete";
-export * from "./core/AutocompleteBase";
```

Comment:
Discussion: Do you think that we should move `AutocompleteBase` and `AutocompleteMultiColumn` to `packages/components/src/core/Autocomplete/components/`, since these two components are implementation detail of `Autocomplete`, so we don't have to have them at the `core` folder like `packages/components/src/core/AutocompleteBase`?

Thank you!

Reply:
Absolutely, that makes a lot of sense. I'll move them into the `Autocomplete/components` directory.